src/inc/clrconfigvalues.h

```diff
@@ -561,6 +561,7 @@ RETAIL_CONFIG_STRING_INFO_EX(EXTERNAL_CORECLR_PROFILER, W("CORECLR_PROFILER"), "
 RETAIL_CONFIG_STRING_INFO_EX(EXTERNAL_CORECLR_PROFILER_PATH, W("CORECLR_PROFILER_PATH"), "CoreCLR only: Specifies the path to the DLL of profiler to load into currently running process", CLRConfig::DontPrependCOMPlus_)
 RETAIL_CONFIG_STRING_INFO_EX(EXTERNAL_CORECLR_PROFILER_PATH_32, W("CORECLR_PROFILER_PATH_32"), "CoreCLR only: Specifies the path to the DLL of profiler to load into currently running 32 process", CLRConfig::DontPrependCOMPlus_)
 RETAIL_CONFIG_STRING_INFO_EX(EXTERNAL_CORECLR_PROFILER_PATH_64, W("CORECLR_PROFILER_PATH_64"), "CoreCLR only: Specifies the path to the DLL of profiler to load into currently running 64 process", CLRConfig::DontPrependCOMPlus_)
+RETAIL_CONFIG_DWORD_INFO_EX(EXTERNAL_CORECLR_DISABLE_REJIT_ON_ATTACH, W("CORECLR_DISABLE_REJIT_ON_ATTACH"), 0, "Disables the ability for profilers to rejit methods on attach.", CLRConfig::DontPrependCOMPlus_)
```

Comment:
What if we named it "EXTERNAL_ProfAPI_RejitOnAttach" and the set default value to 1? I'm not sure we've been consistent over time, but we follow the same pattern in other features such as ReadyToRun or TieredCompilation. (default value 1, positive rather than negative name, user explicitly sets to 0 to disable).
So far we've only used "CORECLR_PROFILER_*" for the variables most profilers need to use when telling the runtime what profiler to load. For everything else we've used the camel case feature switches that start with COMPLUS as seen below.